05-single-hop-route/README.md

```diff
@@ -1,4 +1,14 @@
-## Goal: Check static routing of ICMPv6 packets over a single hop
+Summary
+=======
+
+These tests check the static routing of ICMPv6 packets over a single hop.
+
+| Task no | Description                                                | Automated? | Ease of automation |
+|---------|------------------------------------------------------------|------------|--------------------|
+| 1       | ICMPv6 echo unicast addresess on native (default route)    | No         | Easy               |
+| 2       | ICMPv6 echo unicast addresess on iotlab-m3 (default route) | No         | Easy               |
+| 3       | ICMPv6 echo unicast addresess on native (specific route)   | No         | Easy               |
+| 4       | ICMPv6 echo unicast addresess on iotlab-m3 (static route)  | No         | Easy               |
```

Comment:
Single hop link local is hard but single hop with routes is easy *raises eyebrow*?